### PR TITLE
sd-bus: convenience functions to emit a signal to a destination

### PR DESCRIPTION
**Pull request by eerimoq**
First time contributor.

I have not added any documentation or tests yet. Will do that if the two new functions will be accepted.

### CODE REVIEW

File: src/libsystemd/sd-bus/bus-convenience.c [hunk 1]
```diff
@@ -44,9 +45,43 @@ _public_ int sd_bus_emit_signalv(
                         return r;
         }
 
+        if (destination) {
+                r = sd_bus_message_set_destination(m, destination);
```
Comment (yuwata) [inline src/libsystemd/sd-bus/bus-convenience.c]: If we add the new functions, then I think we should also introduce `sd_bus_message_new_signal_to()`, which also takes destination.

**Comment by yuwata** [comment]
> I have not added any documentation or tests yet. Will do that if the two new functions will be accepted.

Please also update src/libsystemd/libsystemd.sym. Note, at least, this commit will not included in v252. So please add the functions in v253 field.

File: src/libsystemd/sd-bus/bus-convenience.c (hunk 1 above)
Comment (eerimoq) [reply src/libsystemd/sd-bus/bus-convenience.c]: Added it to the PR.

**Comment by yuwata** [comment]
LGTM. Please update documentation and test.

**Comment by eerimoq** [comment]
Feel free to suggest how to test the new functionality. It seems that similar convenience functions are not tested, but I can be missing something.

**Review by yuwata — APPROVED** [review]

**Comment by eerimoq** [comment]
Not sure why `jammy-amd64 — autopkgtest` failed. Any ideas?

**Comment by yuwata** [comment]
> Not sure why `jammy-amd64 — autopkgtest` failed with timeout. Any ideas?

Ubuntu CIs are always unstable. Hopefully, the failure is unrelated to this PR.

**Pull request merged**